bedrock/mozorg/templates/mozorg/moss/index-092018.html

```diff
@@ -325,6 +325,22 @@ <h2 class="moss-section-heading">{{ _('Frequently Asked Questions') }}</h2>
       </p>
     </dd>
 
+    {% if l10n_has_tag('moss_faq_201810') %}
```

Comment:
Depending on @peiying2's answer, if the string changes above are ok, there may be no need for an l10n conditional tag here.

Reply:
The page was being localized, but done so *only* to Hindi and not actively (and hits to the page in Hindi over the past year are anemic).  We're okay to go w/o l10n tags

Reply:
@ejregithub I think we should just check with @peiying2 here before assuming anyting.

Reply:
@alexgibson - I found Lindsay's (owner of the MOSS content) response:  "We discussed this today, and we feel like (given the low numbers) we don't need the localization right now."

She also noted the want to get the content live for Mozfest, though that may've passed considering the timezone diffs . . (Perfect timing before our MozFest session tomorrow- thank you!)

Reply:
Ok, if we can't wait to here from @peiying2 then @amychurchwell please remove the l10n tags.

Reply:
@alexgibson ok, will do!